metadata/extension-readme.md

```diff
@@ -0,0 +1,21 @@
+# ADO Extension for Defender for DevOps
+
+An extension for Azure DevOps that allows [Microsoft Defender for DevOps](https://aka.ms/DfDdocs) to extract metadata from pipelines, such as containers digest ID, container image name etc. That information will then be used within [Microsoft Defender for Cloud](https://aka.ms/mdc) to connect DevOps entities with their related resources running multi-cloud (Azure, AWS, GCP, on-premise).
```

Comment:
Readability: "to connect DevOps entities with their related resources running multi-cloud (Azure, AWS, GCP, on-premise)."
Consider rewording this. It wasn't clear to me exactly what this meant until I read it twice, so I think customers may be confused.
"...to connect DevOps entities with their related cloud resources (Azure, AWS, GCP, on-premises) " might be better. You don't need to say "running multi-cloud" if you include that info in the parentheses anyway.